### PR TITLE
how-to/{deploy,get-started}: make version specific

### PR DESCRIPTION
**Pull request by morgo**
### What is changed, added or deleted? 

Since the docs are now version specific, the download URLs should be too.

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 

All versions

### Check list 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md [hunk 1]
```diff
@@ -29,31 +29,29 @@ Similar to [Homebrew](/how-to/get-started/local-cluster/install-from-homebrew.md
...
+dbdeployer downloads get-unpack --overwrite --delete-after-unpack mysql-5.7.26-linux-glibc2.12-x86_64.tar.gz
 </pre>
 
-Install TiDB using the installed MySQL 5.7 client:
+Install TiDB 3.0 using the installed MySQL 5.7 client:
 <pre>
-wget https://download.pingcap.org/tidb-master-linux-amd64.tar.gz
-dbdeployer unpack tidb-master-linux-amd64.tar.gz --unpack-version=3.0.0
-dbdeployer deploy single 3.0.0 --client-from=5.7.25
+wget https://download.pingcap.org/tidb-v3.0-linux-amd64.tar.gz
```
Comment (morgo) [inline v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: @zhouqiang-cl @kolbe The URL I've used here temporarily is all 3.0 components. Do we have one that is like the previous URL (version 3.0, but only tidb-server)?

File: v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md [hunk 2]
```diff
@@ -29,31 +29,29 @@ Similar to [Homebrew](/how-to/get-started/local-cluster/install-from-homebrew.md
...
 After <a href="https://github.com/datacharmer/dbdeployer">installing DBdeployer</a>, install a MySQL 5.7 client:
 <pre>
-curl -LO https://dev.mysql.com/get/Downloads/MySQL-5.7/mysql-5.7.25-macos10.14-x86_64.tar.gz
-dbdeployer unpack mysql-5.7.25-macos10.14-x86_64.tar.gz
+dbdeployer downloads get-unpack --overwrite --delete-after-unpack mysql-5.7.26-macos10.14-x86_64.tar.gz
 </pre>
 
 Install TiDB using the installed MySQL 5.7 client:
 <pre>
 curl -O https://download.pingcap.org/tidb-master-darwin-amd64.tar.gz
```
Comment (morgo) [inline v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: @zhouqiang-cl @kolbe It looks like we don't have a darwin 3.0 tidb-server only package. Can we create one?

File: v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md (hunk 1 above)
Comment (kolbe) [reply v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: I found https://download.pingcap.org/tidb-release-2.1-linux-amd64.tar.gz but substituting 3.0 gives 404.

File: v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md (hunk 2 above)
Comment (zhouqiang-cl) [reply v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: yes. 3.0 is not exist. @mahjonp PTAL (It builds in Mac mini in beijin office)

Comment (mahjonp) [reply v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: Done, now we can download darwin 3.0 tidb-server using URL: https://download.pingcap.org/tidb-v3.0-darwin-amd64.tar.gz

File: v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md (hunk 1 above)
Comment (kolbe) [reply v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: @zhouqiang-cl @mahjonp should there be tidb-release-3.0-linux-amd64.tar.gz to match the 2.1 package?

```
$ curl -sSL https://download.pingcap.org/tidb-release-2.1-linux-amd64.tar.gz | tar tzvf -
drwxr-xr-x  0 pingcap staff       0 Jun  4 09:07 tidb-release-2.1-linux-amd64/
drwxr-xr-x  0 pingcap staff       0 Jun  4 09:07 tidb-release-2.1-linux-amd64/bin/
-rwxr-xr-x  0 pingcap staff 57154776 Jun  4 03:52 tidb-release-2.1-linux-amd64/bin/tidb-server
```

Comment (mahjonp) [reply v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: @kolbe I have added a daily release job of the release-3.0 branch, and it's ok now.

**Comment by zhouqiang-cl** [comment]
LGTM

**Comment by kolbe** [comment]
It looks like this is pretty outdated now and since the PR is merging from Morgan's branch, maybe someone else needs to re-create it merging from their own branch?

File: v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md (hunk 2 above)
Comment (kolbe) [reply v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: @mahjonp @zhouqiang-cl actually, Morgan was asking for tidb-release-3.0-darwin-amd64.tar.gz (only tidb-server) so the user doesn't have to download all components when they're not needed. Could you please also add that package?

Comment (kolbe) [reply v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md]: Also, it looks like the tidb-v3.0-darwin-amd64.tar.gz package is still at version 3.0.0, even though 3.0.1 has been released.

**Comment by lilin90** [comment]
> It looks like this is pretty outdated now and since the PR is merging from Morgan's branch, and there have been some big structural changes since then anyway. I'll create my own branch and try to resolve conflicts and create a new PR as soon as the builds requested in this one become available.

@kolbe Then close this PR now?

**Comment by lilin90** [comment]
@kolbe I'll close this PR. You can update in your own PR later.

**Pull request closed without merging**